src/components/modules/ModuleCheckout.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  IOColors,
+  IOModuleIDPSavedVSpacing,
+  IOModuleStyles,
+  useIOExperimentalDesign
+} from "../../core";
+import { IOLogoPaymentType, LogoPayment } from "../logos";
+import { VSpacer } from "../spacer";
+import { Body, H6 } from "../typography";
+import { PressableModuleBase } from "./PressableModuleBase";
+
+type ModuleCheckoutProps = { ctaText: string } & (
+  | {
+      paymentLogo?: IOLogoPaymentType;
+      header: string;
+      body: string;
+      isLoading?: false;
+      onPress?: () => void;
+    }
+  | {
+      isLoading: true;
+    }
+);
+
+type CtaOnlyProps = { text?: string };
+
+export const ModuleCheckout = (props: ModuleCheckoutProps) => {
+  if (props.isLoading) {
+    return <LoadingVersion text={props.ctaText} />;
+  }
+  return (
+    <PressableModuleBase onPress={props.onPress}>
+      {props.paymentLogo && <LogoPayment name={props.paymentLogo} />}
+      <View>
+        <H6>{props.header}</H6>
+        <Body>{props.body}</Body>
+      </View>
+      <CTA text={props.ctaText} />
+    </PressableModuleBase>
+  );
+};
+
+const CTA = ({ text }: CtaOnlyProps) => {
+  const isExperimental = useIOExperimentalDesign();
+  const blue: IOColors = isExperimental ? "blueIO-500" : "blue";
+
+  return (
+    <Body color={blue} weight="SemiBold">
+      {text}
+    </Body>
```

Comment:
```suggestion
    <ButtonLink label={text} accessibilityLabel="…" />
```
You can directly use `ButtonLink` and include it in the following `View` to avoid interactive conflicts (for similar behavior, see `Banner` component):
```jsx
<View pointerEvents="none">[…ButtonLink…]</View>
```
Doing so, you can also remove the two lines above (`isExperimental` and ternary condition about color)